docs/user/contacts.md

```diff
@@ -0,0 +1,40 @@
+# Contacts
+## [Bug Report](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=)
+
+A bug is a demonstrable problem that is caused by code in the repository. Bug reports are helpful for developers who maintain the Vald project.
+
+A good bug report shouldn’t leave others needing to ask you constantly for more information. So could you please try to be as detailed as possible in your bug report?
+
+Things to check before reporting the bug:
+- Check your environment
+Bug may occur depending on versions (Go, Docker, Kubernetes, NGT ..etc).
+So please match versions recommended by the Vald project. 
+
+- Identify the bug
+Developers need to reproduce the bug in the developers environment.
+So please Identify the bug and clarify how it occurs.
+Then please clarify the expected behavior too.
+
+- Use the Github issue search
+The bug has already been reported.
+So please check the Github issue before reporting.
+
+## [Proposal](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Ffeature%2C+priority%2Flow%2C+team%2Fcore&template=feature_request.md&title=)
+
+The Vald project’s development process is design-driven. 
+The significant change to the library, or architecture must be first discussed, and sometimes formally documented before it can be implemented.
+The proposal is the process for reviewing a proposal and reaching a decision about whether to accept or decline the proposal.
+After the proposal is accepted or declined, please start implementations.
+
+## [Feature Request](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Ffeature%2C+priority%2Flow%2C+team%2Fcore&template=feature_request.md&title=)
+
+Feature request is welcome.
+Before opening an issue, please make sure your idea fits the scope and goals of the project.
```

Comment:
we need to let user knows the scope and goals before user submit feature request?

Reply:
:memo:
I delete the `the scope and goals`  because the user does not need to know the goal and scope of vald.